include/sot/torque_control/torque-offset-estimator.hh

```diff
@@ -79,7 +79,6 @@ namespace dynamicgraph {
 
         DECLARE_SIGNAL_IN(base6d_encoders,          dynamicgraph::Vector);
         DECLARE_SIGNAL_IN(accelerometer,            dynamicgraph::Vector);
-        DECLARE_SIGNAL_IN(gyroscope,                dynamicgraph::Vector);
```

Comment:
Why did you remove the gyroscope? I think it may be a good idea to keep it just to check that the robot does not rotate during the calibration